lib/Minz/Extension.php

```diff
@@ -157,7 +157,7 @@ private function setType(string $type): void {
 	/** Return the user-specific, extension-specific, folder where this extension can save user-specific data */
 	protected final function getExtensionUserPath(): string {
 		$username = Minz_User::name() ?: '_';
-		return USERS_PATH . "/{$username}/extensions/{$this->getName()}";
+		return USERS_PATH . "/{$username}/extensions/{$this->getEntrypoint()}";
```

Comment:
Will this cause a new entry in the update.php that changes existing paths?

Reply:
Could, if anyone is motivated in doing so. Might be a bit tricky to do safely, though. Another option could be to have a migration code in

https://github.com/FreshRSS/FreshRSS/blob/7aaed6092f09b73b2deb7e32a1fed9b776c0202d/lib/Minz/Extension.php#L83

Reply:
If we expect version jumps, I think it is better method suggested by @Alkarex.
But we can't have a common migration code in `init()`, because it will be overwritten by each extension.
So how about adding a new method that is called just before `init()`?

Reply:
Ideally, the extensions should call the parent method `parent::init()` when overriding. We could fix those we know like I did in https://github.com/FreshRSS/Extensions/pull/228/

Reply:
Ok, then I will implement the migration code in `init()`.

Reply:
It was also necessary to migrate in `handleConfigureAction()` to support the use case of opening the configuration view before enabling the extension.